plugins/base/src/test/java/edu/wpi/first/shuffleboard/plugin/base/recording/serialization/SendableChooserAdapterTest.java

```diff
@@ -1,37 +0,0 @@
-package edu.wpi.first.shuffleboard.plugin.base.recording.serialization;
-
-import edu.wpi.first.shuffleboard.plugin.base.data.SendableChooserData;
-
-import org.junit.jupiter.api.Test;
-
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-
-public class SendableChooserAdapterTest extends AbstractAdapterTest<SendableChooserData> {
```

Comment:
Are you not supporting SenableChooser's anymore?

Reply:
All data recording for FRC data types is currently done by the NetworkTableSourceType to reduce file size. See https://github.com/wpilibsuite/shuffleboard/pull/306#issuecomment-346453399